app/Match.js

```diff
@@ -0,0 +1,18 @@
+export class Match{
+  constructor(Player1,Player2){
```

Comment:
Use camelCase for both variables and property names. Use PascalCase only for class names.